woocommerce/woocommerce
# [Experimental] Product Filters Redesign: Move icons to @woocommerce/icons package

### Changes proposed in this Pull Request:




Currently, the icons for each of the individual filters are inside each filter folder. However, considering that these icons can be used in other places, it would be better for them to be moved to the @woocommerce/icons package.

Closes #48990.



### How to test the changes in this Pull Request:



> [!NOTE]  
> **Pre-requisite**
> - If you're using WC Beta Tester plugin, make sure the `experimental-blocks` feature is enabled (Tools > WCA Test Helper > Features).

1. Navigate to Product Catalog template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filters (Experimental)" in the block inserter or browse the available blocks until you find them. Add it to your content.
5. In the list view, open all of the individual filter blocks.
6. Make sure the icons match the design.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment  Move product filters icons to the `@woocommerce/icons packaged`.

</details>
